pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: seltzer
-version: 0.2.4-alpha
+version: 1.2.4-alpha
```

Comment:
In pre-1.0.0, it's not necessary to make a jump like this.

Just go ahead and bump to `0.3.0`.